frontend/components/SentimentAnalysis2.js

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from "react";
+
+const SentimentAnalysis2 = () => {
+    const [userInput, setUserInput] = useState("");
+    const [submitted, setSubmitted] = useState(false);
+    const [noteData, setNoteData] = useState(null);
+    const [soundData, setSoundData] = useState(null);
+
+    const handleInputChange = (event) => {
+        setUserInput(event.target.value);
+    };
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        fetch(`/api/get_sentiment_analysis_2?text=${userInput}`)
+            .then(response => response.json())
+            .then(data =>{
+                setNoteData(data.note);
+                setSoundData(data.sound);
+            });
+        setSubmitted(true);
+    };
```

Comment:
The following isn't a blocker, but it might be of interest as you continue exploring the React space. In general, React patterns encourage async tasks (fetching from an API, say) to occur inside what they call their "useEffect" hook. You can read more [here](https://reactjs.org/docs/hooks-effect.html). The idea being: perhaps `handleSubmit` simply toggles the `submitted` state to `true`, and the `useEffect` responds to `submitted` changing to `true` with the async fetch. The goal with modern React is to move as aggressively as possible into the 'declarative' programming space, i.e., instead of thinking solely in terms of 'user does X, application responds Y', you write your code such instead as 'when data does X, app responds Y.' It's a subtle distinction in this case, so please feel free to keep the code as is, but you may find it interesting to read more about the `useEffect` hook in particular.